### PR TITLE
Add configuring MI when Analytics running in different VMs

### PR DESCRIPTION
**Pull request by ramindu90**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ Let's start the servers in the given order.
 !!! Warning
     Be sure to [configure the SSL certificates](../../setup/observability/setting-up-classic-observability-deployment/#configure-ssl-certificates) before starting the Micro Integrator.
 
+#### Configuring the Micro Integrator to Publish Data to Analytics Server
+Analytics publishing can be configured in `<EI_ANALYTICS_HOME>/conf/carbon.xml` in 'Analytics' section.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md:69]: this needs to be MI_HOME ? and we need to give instructions on how to configure this in toml ?

File: en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md [hunk 2]
```diff
@@ -65,6 +65,19 @@ Let's start the servers in the given order.
 !!! Warning
     Be sure to [configure the SSL certificates](../../setup/observability/setting-up-classic-observability-deployment/#configure-ssl-certificates) before starting the Micro Integrator.
 
+#### Configuring the Micro Integrator to Publish Data to Analytics Server
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md:68]: @ramindu90 - shall we add this section to the 'Setting up classic observability deployment' doc?
according to how our doc is structured, we have all the 'setting up content in one place..

File: en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md [hunk 3]
```diff
@@ -107,6 +119,62 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
...
+    ![all events to all servers](../../assets/img/ei-analytics/ob-all-events-to-all-servers.png)
+    
+eg: `{tcp://Analytics-1>:<port>}, {tcp://Analytics-2>:<port>}, {tcp://<Analytics-3>:<port>}`
+
+sample `{tcp://10.100.2.32:7611},{ tcp://10.100.2.33:7611}, {tcp://10.100.2.34:7611}`
+
+#### Failover configuration
+
+When using the failover configuration in publishing events to Analytics, events are sent to multiple Analytics servers in a sequential order based on priority. 
+You can specify multiple Analytics servers so that events can be sent to the next server in the sequence in a situation where they were not successfully sent to the first server.
```
Comment (RukshiW) [inline en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md:169]: Change "in the sequence" to "in the specified sequence".

File: en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md [hunk 4]
```diff
@@ -107,6 +119,62 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
...
+eg: `{tcp://Analytics-1>:<port>}, {tcp://Analytics-2>:<port>}, {tcp://<Analytics-3>:<port>}`
+
+sample `{tcp://10.100.2.32:7611},{ tcp://10.100.2.33:7611}, {tcp://10.100.2.34:7611}`
+
+#### Failover configuration
+
+When using the failover configuration in publishing events to Analytics, events are sent to multiple Analytics servers in a sequential order based on priority. 
+You can specify multiple Analytics servers so that events can be sent to the next server in the sequence in a situation where they were not successfully sent to the first server. 
+In the scenario depicted in the above image, the events are first sent to Analytics-1. 
+If it is unavailable, then events will be sent to Analytics-2.
```
Comment (RukshiW) [inline en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md:171]: Change "will be" to "are".

File: en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md [hunk 5]
```diff
@@ -107,6 +119,62 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
...
+
+sample `{tcp://10.100.2.32:7611},{ tcp://10.100.2.33:7611}, {tcp://10.100.2.34:7611}`
+
+#### Failover configuration
+
+When using the failover configuration in publishing events to Analytics, events are sent to multiple Analytics servers in a sequential order based on priority. 
+You can specify multiple Analytics servers so that events can be sent to the next server in the sequence in a situation where they were not successfully sent to the first server. 
+In the scenario depicted in the above image, the events are first sent to Analytics-1. 
+If it is unavailable, then events will be sent to Analytics-2. 
+If Analytics-2  is also unavailable, then the events will be sent to Analytics-3.
```
Comment (RukshiW) [inline en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md:172]: Change "will be" to "are".

File: en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md [hunk 6]
```diff
@@ -107,6 +119,62 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
...
+    ![fail over](../../assets/img/ei-analytics/ob-fail-over.png)
+
+eg: `tcp://<Analytics-1>:<port>|tcp://<Analytics-2>:<port>|tcp://<Analytics-3>:<port>`
```
Comment (RukshiW) [inline en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md:175]: Change "eg:" to "e.g.,".

**Review by RukshiW — COMMENTED** [review]
Requesting some changes.

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**